### ISSUE TITLE
Large volume spacing

### ISSUE THREAD

**Issue by sheryjoe** [SCIInstitute/Seg3D#281]
Seg3D can't load the spacing correctly (from CreateLargeVolume tool output) and I can not changed from the transform tool. Below is where the data I am using can be found.

/home/sci/shireen/Public/ForSeg3D/

**Comment by mildewey** [comment]
We've very recently made a change that may have improved/fixed a spacing issue in regards to dicom files. Is this what you are running into?

What version of Seg3D are you using?

**Comment by jessdtate** [comment]
Just to add a little more information. The spacing was loading correctly, but was set incorrectly.  Currently the only way to change this is to rerun the large volume creator tool and change the spacing.  We should be able to modify the transform tool to work with large volume formats.

**Comment by sheryjoe** [comment]
Hi,

A quick update. I have been able to use CreateLargeVolume tool with isotropic voxel spacing and load that into Seg3D (latest version) and it loaded correctly with the exact spacing that I used for CreateLargeVolume too.

thanks and best regards
Shireen


------------------------------------------
Shireen Elhabian, M.Sc., Ph.D.
Research Assistant Professor
School of Computing
Scientific Computing and Imaging Institute
University of Utah
WEB 2815
72 Central Campus Drive, Salt Lake City, UT, 84112
Phone (801) 587-3206
Fax (801) 585-6513
Home Page: http://www.sci.utah.edu/~shireen <http://www.sci.utah.edu/~shireen>
